low-frame-rate-codec.patch

```diff
@@ -0,0 +1,1265 @@
+diff --git a/nemo/collections/tts/models/audio_codec.py b/nemo/collections/tts/models/audio_codec.py
```

Comment:
This file needs to be removed from the PR

Reply:
Done